www/js/app.js

```diff
@@ -98,6 +98,15 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
+ 
+    var darkPreference = window.matchMedia("(prefers-color-scheme : dark");
+    darkPreference.addEventListener("change", e=> {
+            if (params.appTheme == "auto") {
+            let newColorScheme = e.matches ? "dark_invert" : "light";
```

Comment:
Use `var` instead of `let`, again for compatibility reasons.

Reply:
Since you will need to handle this variable in `uiUtil`, you will need to use `params.newColourScheme` (this is because the params object is global and is available outside `app.js`, whereas `newColorScheme` would be available only in `app.js`).

Reply:
Ohk Sir.